docs/tutorials/index.rst

```diff
@@ -11,4 +11,5 @@ Tutorials
    ../cli.rst
    first_steps_config
    context_manager
-   wrapup
\ No newline at end of file
+   wrapup
```

Comment:
Oh, sorry, my bad. I don't know why I thought you had added it yourself. Sorry for the confusion.